src/components/contact/ContactSocialMedia.vue

```diff
@@ -0,0 +1,218 @@
+<script>
+import { defineComponent } from "vue";
+import BaseButton from "@/components/global/BaseButton.vue";
+
+export default defineComponent({
+  name: "ContactSocialMedia",
+  components: { BaseButton },
+});
+</script>
+
+<template>
+  <BaseButton
+    href="#"
+    target="blank"
+    rel="noopener"
+    class="contact__page-button"
+  >
+    <span class="visually-hidden">Twitter (opens in new tab)</span>
+    <svg
```

Comment:
Is this the same icon as the one in assets?

Reply:
Did you have any particular issues using it here?

Reply:
The idea was to provide some label to the svg icons for the screen readers.

First I imported svg as a component and placed it inside anchor tag - but there was no label for screen readers.

Then I tried to use img tag where I could use alt attribute as a label, but setting scr as a path to the svg file or imported svg component didn’t work - I would get broken icon on the page.

That’s why I decided to use svg tags with attributes aria-hidden="true"  focusable="false" and add span element with a label for screen readers. Not sure about this solution, found it in the net.

Yes, all the svg icons are stored in the assets.

Reply:
I believe what you're looking for is `aria-label` that is used to provide a description on interactive elements with no text content like icons links :)  https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8#:~:text=The%20aria%2Dlabel%20attribute%20provides,used%20instead%20of%20aria%2Dlabel%20.

In regards to the svg using them as components after importing with `?component` should give you the same results, if I'm not mistaken you can set attributes on it if needed.